linux_os/guide/services/ssh/ssh_server/sshd_disable_pubkey_auth/rule.yml

```diff
@@ -0,0 +1,30 @@
+documentation_complete: false
+
+title: 'Disable PubkeyAuthentication Authentication'
+
+description: |-
+    Unless needed, SSH should not permit extraneous or unnecessary
+    authentication mechanisms. To disable PubkeyAuthentication authentication, add or
+    correct the following line in the <tt>/etc/ssh/sshd_config</tt> file:
+    <pre>PubkeyAuthentication no</pre>
+
+rationale: |-
+    PubkeyAuthentication authentication is used to provide additional authentication mechanisms to
+    applications. Allowing PubkeyAuthentication authentication through SSH allows users to
+    generate their own authentication tokens, increasing the attack surface of the system.
+
+ocil_clause: 'it is not disabled'
+
+ocil: |-
+    To check if PubkeyAuthentication is disabled or set correctly, run the following
+    command:
+    <pre>$ sudo grep PubkeyAuthentication /etc/ssh/sshd_config</pre>
+    If configured properly, output should be <pre>no</pre>
+
+template:
+    name: sshd_lineinfile
+    vars:
+        missing_parameter_pass: 'false'
+        parameter: PubkeyAuthentication
+        rule_id: sshd_disable_pubkey_auth
+        value: 'no'
```

Comment:
> I'm not sure how to build remediation scripts or documentation....

It is good that you have already added this data here. The new templating mechanism is not yet enabled by default.
The CSV files are stills the default. 

Also add the data here: 
https://github.com/ComplianceAsCode/content/blob/master/shared/templates/csv/sshd_lineinfile.csv

Reply:
Done, I'll push a new version